### PR TITLE
Backport changelog fixups from http://cl/229787474

### PR DESCRIPTION
**Pull request by rsgowman**

### CODE REVIEW

File: firebase-firestore/CHANGELOG.md [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
+
+# 18.0.0
 - [changed] The `timestampsInSnapshotsEnabled` setting is now enabled by
-  default so timestamp fields read from a `DocumentSnapshot` will be returned
-  as `Timestamp` objects instead of `Date`. Any code expecting to receive a
-  `Date` object must be updated.
-- [feature] Custom objects (POJOs) can now be passed as a field value in
-  update(), within `Map<>` objects passed to set(), in array transform
-  operations, and in query filters.
+  default. Timestamp fields taht read from a `DocumentSnapshot` are now
```
Comment (wilhuff) [inline firebase-firestore/CHANGELOG.md]: s/taht/that/

File: firebase-firestore/CHANGELOG.md [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
+  returned as `Timestamp` objects instead of `Date` objects. This is a breaking
+  change; developers must update any code that expects to receive a `Date`
+  object. See https://firebase.google.com/docs/reference/android/com/google/firebase/firestore/FirebaseFirestoreSettings.Builder.html#setTimestampsInSnapshotsEnabled(boolean) for more details.
```
Comment (wilhuff) [inline firebase-firestore/CHANGELOG.md]: Seems like this is something we should port into the iOS release notes too?

File: firebase-firestore/CHANGELOG.md [hunk 3]
```diff
@@ -1,17 +1,19 @@
...
+- [feature] Custom objects (POJOs) can now be passed in several ways: as a
+  field value in update(), within `Map<>` objects passed to set(), in array
```
Comment (wilhuff) [inline firebase-firestore/CHANGELOG.md]: Do "update()" and "set()" need backticks?

File: firebase-firestore/CHANGELOG.md [hunk 4]
```diff
@@ -1,17 +1,19 @@
...
+  transform operations, and in query filters.
 - [feature] DocumentSnapshot.get() now supports retrieving fields as
-  custom objects (POJOs) by passing a Class<T> instance, e.g.
+  custom objects (POJOs) by passing a Class<T> instance, e.g.,
```
Comment (wilhuff) [inline firebase-firestore/CHANGELOG.md]: Does "DocumentSnapshot.get()" need backticks? Does "Class<T>" need them too?

File: firebase-firestore/CHANGELOG.md (hunk 1 above)
Comment (rsgowman) [reply firebase-firestore/CHANGELOG.md]: Done.

File: firebase-firestore/CHANGELOG.md (hunk 2 above)
Comment (rsgowman) [reply firebase-firestore/CHANGELOG.md]: Yep. I'll adjust the other ports shortly.

File: firebase-firestore/CHANGELOG.md (hunk 3 above)
Comment (rsgowman) [reply firebase-firestore/CHANGELOG.md]: "Need" might be too strong, but yes, they should be there. Done.

File: firebase-firestore/CHANGELOG.md (hunk 4 above)
Comment (rsgowman) [reply firebase-firestore/CHANGELOG.md]: Done.

File: firebase-firestore/CHANGELOG.md (hunk 2 above)
Comment (rsgowman) [reply firebase-firestore/CHANGELOG.md]: https://github.com/firebase/firebase-ios-sdk/pull/2286
https://github.com/firebase/firebase-js-sdk/pull/1484

**Pull request merged**